packages/@aws-cdk/aws-eks/lib/cluster.ts

```diff
@@ -232,16 +336,19 @@ export interface ClusterProps extends ClusterOptions {
    * - `addRoleMapping`
    * - `addUserMapping`
    * - `addMastersRole` and `props.mastersRole`
+   * - `endpointAccess`
    *
    * If this is disabled, the cluster can only be managed by issuing `kubectl`
    * commands from a session that uses the IAM role/user that created the
    * account.
    *
-   * _NOTE_: changing this value will destoy the cluster. This is because a
+   * _NOTE_: changing this value will destroy the cluster. This is because a
    * managable cluster must be created using an AWS CloudFormation custom
    * resource which executes with an IAM role owned by the CDK app.
    *
+   *
    * @default true The cluster can be managed by the AWS CDK application.
+   * @deprecated Omit this property as kubectl will be enabled to all clusters.
```

Comment:
Let's leave this for later okay? I think we need to discuss further

Reply:
👍